client/public/index.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <meta charset="UTF-8" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css">
```

Comment:
We tend to prefer NPM packages that we can yarn install over CDN links for consistency with dependency management (better to see all dependencies in the package json rather than having some as separate CDNs)

Reply:
Following fontawesome's yarn install directions caused the whole app to not work.  This is from the twitter feed assignment.